src/commands.ts

```diff
@@ -352,6 +358,37 @@ export function expandMacro(ctx: Ctx): Cmd {
   };
 }
 
+export function explainError(ctx: Ctx): Cmd {
+  return async () => {
+    const { document, position } = await workspace.getCurrentState();
+    if (!isRustDocument(document)) return;
+
+    const diagnostic = ctx.client.diagnostics?.get(workspace.uri)?.find((diagnostic) => isInRange(diagnostic.range, position));
+
+    if (diagnostic && diagnostic?.code) {
+      const explaination = spawnSync('rustc', ['--explain', `${diagnostic!.code!}`], { encoding: 'utf-8' }).stdout;
+
+      const docs: Documentation[] = [];
+      let isCode = false;
+      for (const part of explaination.split('```\n')) {
+        docs.push({ content: part, filetype: isCode ? 'rust' : 'markdown' });
+        isCode = !isCode;
+      }
+
+      if (ctx.config.explainInPopup) {
+        const factory: FloatFactory = new FloatFactory(workspace.nvim, workspace.env);
+        await factory.create(docs);
```

Comment:
Use `show`, because `create` is deprecated https://github.com/neoclide/coc.nvim/blob/41d77ab2ca842a777184b7b9f18d532357b87eba/src/model/floatFactory.ts#L108

Reply:
That is `master` tho, not the current latest release, right? On the current release `0.0.79`,  it's still the only option (show does not even exist)
for reference: https://github.com/neoclide/coc.nvim/blob/4f40c16a15336b589b1b5b509df4e00300d755eb/src/model/floatFactory.ts#L120

Reply:
Yes, `show` method was added in latest release.